### DISCUSSION TITLE
Update mpv version in flatpak package

### DISCUSSION THREAD

**Discussion by OCASM** [smplayer-dev/smplayer#731] [Ideas]
The current version of mpv is 0.35.1. The version used in the flatpak is 0.34.1 which is over a year old. It causes issues with audio not playing in some youtube videos.

**Comment by smplayer-dev** [comment]
You can find here a new flatpak with mpv 0.35.1:
https://github.com/smplayer-dev/smplayer/releases/tag/v2023.06.12

**Reply by OCASM** [reply]
Thanks!

Any reason why the flathub repo is not being updated?

**Reply by smplayer-dev** [reply]
It will be updated when a new version is released.
